TerminologService/valueSetToRoots.py

```diff
@@ -22,7 +23,23 @@ def expand_value_set(url: str, onto_server: str = TERMINOLOGY_SERVER_ADDRESS):
     if '|' in url:
         url = url.replace('|', '&version=')
     term_codes = SortedSet()
-    response = requests.get(onto_server + f"ValueSet/$expand?url={url}", cert=(SERVER_CERTIFICATE, PRIVATE_KEY))
+    version = MAPPING_ONTO_VERSION.get('canonical_address')
+    onto_server = TERMINOLOGY_SERVER_ADDRESS
+
+    if version:
+        canonical_address = f"ValueSet/$expand?url={url}&system-version={version}"
+    else:
+        canonical_address = f"ValueSet/$expand?url={url}"
+        logging.debug(f"No version found for canonical address: {canonical_address}")
```

Comment:
The log output is not in line with the implementation. You never look up {canonical_address} but "canonical_address"

Reply:
Would that be a better logging message which clarifys the purpose of the error?
`logging.debug(f"No version was provided.Using canonical address: {canonical_address}")`

Reply:
yes